scripts/install_operator.sh

```diff
@@ -29,7 +29,7 @@ EOF
 fi
 
 kubectl apply --namespace "${NAMESPACE}" -f ${SPECS_DIR}/spark-driver-rbac.yaml
-# Expose Spark Operator metrics service
-kubectl apply --namespace "${NAMESPACE}" -f ${SPECS_DIR}/spark-operator-service.yaml
-# Create ServiceMonitor (see prometheus-operator docs) for Spark
+
+# Create ServiceMonitor (see prometheus-operator docs) for Spark.
+# TODO: make the service-monitor.yaml part of kudo-spark-operator once the issue https://github.com/kudobuilder/kudo/issues/913 will be fixed.
```

Comment:
can we please make this information available in `monitoring.md` until this bug is not fixed?

Reply:
Ok, make sense

Reply:
has it been added to `monitoring.md`? can't find it there

Reply:
https://github.com/mesosphere/kudo-spark-operator/pull/17/files#diff-fa49808299ade494be7376593e58101fR12-R30